addons/hr/migrations/13.0.1.1/post-migration.py

```diff
@@ -15,9 +19,16 @@ def convert_image_attachments(env):
             ('res_id', '!=', False),
         ])
         for attachment in attachments:
-            # for not having dangling attachments
-            attachment.res_field = "image_1920"
```

Comment:
why delete this?

Reply:
Because it's not following the preservation principle and it's not done in the rest. There's no extra space used, being the same file with the same hash.